onnxscript/rewriter/no_op.py

```diff
@@ -24,7 +24,7 @@ def div_by_1(x):
 
 
 # Replacement
-def identity(x):
+def identity(op, x):
```

Comment:
Should we add type?

Reply:
Once things settle down (and there is agreement on the type, which is a newly added one ... eg., should it be a `RewriterContext` as proposed or an `OpBuilder` ... I think for long run, context is better if we want to add more info in the future. Should the object still be called `op` (more aligned with the onnxscript usage, and existing use)?

Reply:
That's a good question. I am happy to explore different options later